projects/plugins/jetpack/extensions/blocks/subscriptions/controls.js

```diff
@@ -220,7 +220,7 @@ export default function SubscriptionControls( {
 				initialOpen={ false }
 				className="wp-block-jetpack-subscriptions__displaypanel"
 			>
-				{ subscriberCount > 1 && (
+				{ subscriberCount >= 1 && (
```

Comment:
This was done on purpose to show only for 2+ subscribers

Reply:
> > Reworded to "You have X subscribers." because "X readers are subscribed." feels unnatural.
> 
> Keep as is in the original design for now.
> 
> > Removed the underlining of X subscribers. (Okay to keep things simple?)
> 
> Same as above!
> 
> > I moved the subscriber count notice under Settings.
> 
> Please, move it back to where it was — ideally it shows below the description, but that is a bit awkward to achieve right now. In settings it's way too buried.

Alright, will move things back to the top under Styles.

Reply:
> This was done on purpose to show only for 2+ subscribers

And this one as well!

Reply:
Once we merge we should look at a better fallback (like something that encourages you to invite people, etc), but one step at a time.

Reply:
Okay, park it for next.